boards/norik/octopus_io_board/doc/index.rst

```diff
@@ -0,0 +1,143 @@
+
+The GPIO pin header provides 7 I/O pins, which can be used as digital input/output. Some of them also serve as chip selects for SPI peripherals.
+
+Power supply
+============
+
+The Octopus IO-Board can be powered from the following sources:
+
+* USB-C connector
+* Solar cell
+* Alkaline battery
+* Li-Po battery
+
+When powered from USB-C or solar cell, the board can charge the Li-Po battery. The battery voltage can be monitored using ADC which can
+provide information about the battery State of charge (SOC).
+
+When powered from alkaline battery, the user needs to set switch SW1 to ALK position. This ensures that the Li-Ion battery is not charged from the alkaline battery.
+
+The board has a built-in LDO regulator that is used to power the Octopus SoM and peripherals. The EN2 pin can be used to enable/disable output 2 of the LDO regulator.
+This can be used to power off peripherals to save power when they are not needed.
+
+The board also has multiple built-in test points for measuring current consumption of the board, which enables the user to measure and optimize the power consumption of the board.
+
+Programming and Debugging
+*************************
+
+Norik Octopus IO-Board can be programmed and debugged using the Tag-Connect TC2030-IDC 6-pin connector or 6-pin SWD pinheader.
+
+Building an application
+=======================
+
+In most case you'll need to use ``octopus_io_board/nrf9160/ns`` target for building examples.
+Some examples don't require non secure mode and can be built with ``octopus_io_board/nrf9160`` target.
+
+Flashing
+========
+Refer to the instruction in the :ref:`nordic_segger` page to install and
+configure all the necessary software.
+
+Here is an example for the Hello World application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom /dev/<tty_device> 115200
+
+Replace <tty_device> with the port where the Octopus IO-Board can be found. For example, under Linux, /dev/ttyACM0.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: octopus_io_board/nrf9160
+   :goals: build flash
+
+To build and flash the application in non-secure mode, use the following command:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: octopus_io_board/nrf9160/ns
+   :goals: build flash
+
+Debugging
+=========
+Refer to the instruction in the :ref:`nordic_segger` page for information on
+debugging.
+
+Testing the on-board LED
+========================
+Use the :zephyr:code-sample:`blinky` to test the on-board LED. Build and flash the example to make sure Zephyr is running correctly on your board.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/blinky
+   :board: octopus_io_board/nrf9160
+   :goals: build flash
+
+References
+**********
+
+.. target-notes::
+
+.. Octopus IO-Board product page: https://www.norik.com/2024/09/16/octopus-io-board/
```

Comment:
In this context, `..` comments out the line, I don't think that was your intent :)

Reply:
Yeah, it's an oversight... Fixed

Reply:
you probably forgot to push the fix?

Reply:
Yes, my bad, sorry!

Reply:
@fplohl for the record the reason I was confused is that none of these external targets are *actually* referenced in the document, so the section is empty

https://builds.zephyrproject.io/zephyr/pr/74523/docs/boards/norik/octopus_io_board/doc/index.html#references

It might make sense to update the text of the intro so that it actually uses the targets, at which point the `.. target-notes` directive will properly list them in the References section. Hope this helps!

See also https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#cross-referencing-external-content

Reply:
I understand. Updated the missing references in the text, should be fixed now

Reply:
perfect, thx!